values.yaml

```diff
@@ -0,0 +1,80 @@
+USER-SUPPLIED VALUES:
```

Comment:
mb we put this cfg to `charts/platform-reports/values-dev.yaml`, or this one is used once to deploy reports to the dev? mb let's ignore it?

Reply:
sorry, forgot to remove